site/content/en/docs/Third Party Content/examples.md

```diff
@@ -25,4 +25,16 @@ You will need to download the Minetest client separately to play.
 * [Quilkin](https://github.com/googleforgames/quilkin) is a non-transparent UDP proxy specifically designed for use with large scale multiplayer dedicated game server deployments, to ensure security, access control, telemetry data, metrics and more.
 * [Quilkin with Agones](https://github.com/googleforgames/quilkin/tree/main/examples) is an example of running [Xonotic](https://xonotic.org/) with Quilkin on an Agones cluster, utilising either [the sidecar integration pattern](https://github.com/googleforgames/quilkin/tree/main/examples/agones-xonotic-sidecar) or via the the [Quilkin xDS Agones provider](https://github.com/googleforgames/quilkin/tree/main/examples/agones-xonotic-xds) with a TokenRouter to provide routing and access control to the allocated GameServer instance.
 
-You will need to download the Xonotic client to interact with the demo.
\ No newline at end of file
+You will need to download the Xonotic client to interact with the demo.
+
+## Shulker
+
+- [Shulker](https://github.com/jeremylvln/Shulker) is a Kubernetes operator for managing complex and dynamic Minecraft
+  infrastructure, including game servers and proxies.
+- Under the hood, it makes use of Agones to manage the pods lifecycles through `Fleet` and `GameServer` resources.
+- The end user does not manage Agones resources directly but rather simplified abstractions provided by Shulker
+  dedicated for Minecraft workloads.
+- It achieves high availability and density Minecraft infrastructures as long as Agones is just as much.
```

Comment:
I took example from the other above but sure, I can work simplify a bit :)

Reply:
Thanks!

i started with looking at

> It achieves high availability and density Minecraft infrastructures as long as Agones is just as much.

Because I wasn't sure what you were trying to say here, and then I figured the statement 

> Under the hood, it makes use of Agones to manage the pods lifecycles through `Fleet` and `GameServer` resources.

Was probably redundant (since this is posted on Agones after all!), and once I did that, I figured we could make it a simpler statement.

That does lead me to one tweak to the above suggestion that may be even better:

```suggestion
- [Shulker](https://github.com/jeremylvln/Shulker) is a Kubernetes operator for managing complex and dynamic Minecraft infrastructure at scale, including game servers and proxies. 
- It builds on top Agones `GameServer` and `Fleet` primitives to provide simplified abstractions specifically tailored to orchestrating Minecraft workloads. 
```

Howzat?

Reply:
I really like the suggestion, fixed :)